airbyte-integrations/bases/base-normalization/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM fishtownanalytics/dbt:0.18.1
```

Comment:
Should we use the `airbyte/base-airbyte-protocol-python` and do a pip install of dbt instead?

Reply:
fair enough.